src/DynamoDbModel.php

```diff
@@ -86,11 +86,33 @@ protected function setupDynamoDb()
 
     public function save(array $options = [])
     {
-        if (!$this->getKey()) {
-            $this->fireModelEvent('creating');
+        $create = !$this->exists;
+        if ($create && $this->fireModelEvent('creating')  === false) {
+            return false;
         }
```

Comment:
OCD alert 😄. Can we add one empty line after `}` so it looks more consistent? Thanks!